File: Programmers/JAVA/lv1/lv1_최소직사각형.java
https://programmers.co.kr/learn/courses/30/lessons/86491?language=java

class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        int row = 0, col = 0, tmp = 0;
        
        for(int i = 0; i < sizes.length; i++){
            if(sizes[i][0] < sizes[i][1]){
                tmp = sizes[i][0];
                sizes[i][0] = sizes[i][1];
                sizes[i][1] = tmp;
            }
            if(row < sizes[i][0]) row = sizes[i][0];
            if(col < sizes[i][1]) col = sizes[i][1];
        }
        
        answer = row * col;
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_최소직사각형.cs
https://school.programmers.co.kr/learn/courses/30/lessons/86491?language=csharp

using System;

public class Solution
{
    public int solution(int[,] sizes)
    {
        int answer = 0;

        int w = 0, h = 0;
        for (int i = 0; i < sizes.GetLength(0); i++)
        {
            w = Math.Max(w, Math.Min(sizes[i, 0], sizes[i, 1]));
            h = Math.Max(h, Math.Max(sizes[i, 0], sizes[i, 1]));
        }
        answer = w * h;

        return answer;
    }
}